cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -184,6 +185,15 @@ func getAPIServerCommand(cfg *kubeadmapi.ClusterConfiguration, localAPIEndpoint
 		}
 	}
 
+	// TODO: The following code should be remvoved after dual-stack is GA.
+	// Note: The user still retains the ability to explicitly set feature-gates and that value will overwrite this base value.
+	enabled, present := cfg.FeatureGates[features.IPv6DualStack]
+	if present && enabled {
+		defaultArguments["feature-gates"] = features.IPv6DualStack + "=true"
+	} else if present && !enabled {
```

Comment:
why not just `else` ? (the remaining case is `!present`)

Reply:
[1] Ideally, we would like to cover the `present==true` case. So something like this should work:
```golang
if present {
        defaultArguments["feature-gates"] = fmt.Sprintf("%s=%t", features.IPv6DualStack, enabled)
}
```

Reply:
Done!